lib/tile.ml

```diff
@@ -0,0 +1 @@
+type tile = char
```

Comment:
Add points field for type tile in addition to the occupying letter (This might be outside the scope for Beta)

Reply:
I think this is outside the scope of our alpha, which is why I ended up removing references to this type in #4

Reply:
Ah yeah okay my next question was wondering if we needed a curr_tile in game_state. sounds good